acceptancetests/assess_deploy_lxd_profile.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+"""Assess when deploying with LXD Profile charm using the 'deploy' command."""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from jujucharm import (
+    local_charm_path,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+    JujuAssertionError,
+)
+
+__metaclass__ = type
+
+log = logging.getLogger("assess_lxdprofile_deploy_charm")
+charm_bundle = 'lxd-profile.yaml'
+
+def setup(client, series):
+    charm_sink = local_charm_path(
+        charm='charms/lxd-profile',
+        juju_ver=client.version,
+        series=series,
+        repository=os.environ['JUJU_REPOSITORY'])
+    _, primary = client.deploy(charm_sink, series=series)
+    client.wait_for(primary)
+
+def assess_juju_lxdprofile_machine(client, args):
+    """ Tests juju status
+
+    Verify with juju status
+    """
+    lxdprofile_machine_verify(client, "juju-{}-lxd-profile-0".format(client.model_name))
+
+def lxdprofile_machine_verify(client, profilename):
```

Comment:
Do we want to verify that the profile got written as well?

Reply:
This is verified using the status output. Considering how this works under (gets the information from the lxc API directly) this should be a non-issue.